keps/sig-api-machinery/2558-publish-version-openapi/README.md

```diff
@@ -123,14 +123,14 @@ feature gate associated with the field.
 This information is derived from comment tag on API fields with the format:
 
 ```
-// +lifecycle:kubernetes:minVersion=<k8s-version>,status=<prelease-status>,featureGate=<featuregate-name>
+// +lifecycle:component=<component-name>,minVersion=<version>,status=<prerelease-status>,featureGate=<featuregate-name>
 ```
 
 The following Kubernetes-specific constraints are validated by a linter:
 
-- The `+lifecycle:kubernetes:minVersion=<k8s-version>` tag denotes that the field is
-in the prerelease since the specified Kubernetes version. It must map to a
-Kubernetes release (e.g. `v1.20`).
+- The `+lifecycle:component=` value is always `kubernetes`.
```

Comment:
clarify as kubernetes = github.com/kubernetes/kubernetes, not github.com/kubernetes-sigs/....

Reply:
Done, PTAL.